calls.go

```diff
@@ -33,9 +34,32 @@ func calls() cli.Command {
 			{
 				Name:      "list",
 				Aliases:   []string{"l"},
-				Usage:     "list all calls for specific app / route route is optional",
-				ArgsUsage: "<app> [route]",
+				Usage:     "list all calls for the specific app. Route is optional",
+				ArgsUsage: "<app>",
 				Action:    c.list,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "path",
+						Usage: "function's path",
+					},
+					cli.StringFlag{
+						Name:  "cursor",
+						Usage: "pagination cursor",
+					},
+					cli.StringFlag{
+						Name:  "from-time",
+						Usage: "'start' timestamp",
+					},
+					cli.StringFlag{
+						Name:  "to-time",
+						Usage: "'stop' timestamp",
+					},
+					cli.Int64Flag{
+						Name:  "per-page",
+						Usage: "number of calls to return",
+						Value: int64(30),
```

Comment:
just omit this imo, the server will sanely default to 30 if <= 0 is provided and we can change that in the future and have it fall out here.

Reply:
There's a small problem, `cli.Int64Flag` will set the default value to 0 and will show it to users, so, keeping this default will at least be good for UX. That's why I don't want to move it away.

Reply:
makes sense